applications/peripheral_sensor_node/README.rst

```diff
@@ -0,0 +1,268 @@
+The :file:`_def` files are used by the nRF Peripheral Sensor node application modules and :ref:`lib_caf` modules.
+
+Configuration files are defined in the :file:`applications/peripheral_sensor_node/configuration/nrf54l15dk_nrf54l15_cpuapp` directory.
+
+Advertising configuration
+-------------------------
+
+If a given build type enables Bluetooth, the :ref:`caf_ble_adv` is used to control the Bluetooth advertising.
+This CAF module relies on :ref:`bt_le_adv_prov_readme` to manage advertising data and to scan response data.
+The nRF Peripheral Sensor node application configures the data providers in the :file:`src/util/Kconfig` file.
+By default, the application enables a set of data providers available in the |NCS|.
+If the Noridc UART Service (NUS) is enabled in the configuration, and the Bluetooth local identity in use has no bond, the application also adds a custom provider that appends UUID128 of NUS to the scan response data.
+
+Shield Configuration
+====================
+
+To ensure that the measurement is triggered by an interrupt from the ``ADXL362`` sensor, you need to set a jumper on the shield board.
+Place a jumper on header P3 to connect the IRQ_U3 signal.
+This configuration allows the application to respond to interrupts generated by the sensor.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`applications/peripheral_sensor_node`
+
+.. include:: /includes/application_build_and_run.txt
+
+Complete the following steps to program the sample:
+
+1. Go to the sample directory.
+#. Open the command line terminal.
+#. Run the following command to build the application code:
+
+   .. code-block:: console
+
+      west build -b nrf54l15pdk/nrf54l15/cpuapp -- -DSHIELD=pca63565
+
+   Alternatively, you can use the twister test case by running the following command:
+
+   .. code-block:: console
+
+      west build -b nrf54l15pdk/nrf54l15/cpuapp -T applications.peripheral_sensor_node .
+
+#. Program the application to the kit:
+
+   .. code-block:: console
+
+      west flash
+
+If you are using the |nRFVSC|, specify the ``-DSHIELD=pca63565`` argument in the **Extra Cmake arguments** field when `setting up a build configuration <How to work with build configurations_>`_.
+
+Application logs
+================
+
+The application provides logs through the UART of the PDK.
+
+.. _nrf_peripheral_sensor_node_app_testing:
+
+Testing with the PDK
+====================
+
+After programming the application, perform the following steps to test the nRF Peripheral Sensor node application on the Preview Development Kit (PDK):
+
+1. Turn on the PDK.
+   The application starts Bluetooth advertising.
+#. Start the `nRF Connect for Mobile`_ application on your smartphone or tablet.
+#. Connect to the device from the application.
+   When connected, go to more options and click :guilabel:`Request MTU`.
```

Comment:
```suggestion
   When connected, go to :guilabel:`more options` and click :guilabel:`Request MTU`.
```

Reply:
If it is a gui label, that should be highlighted (also with the correct capitalization)